File: Java/ejercicio1Fich/Ejercicio1.java
package ejercicio1Fich;

import java.io.*;

/**
 * Escribe un programa que guarde en un fichero con nombre primos.dat los
 * números primos que hay entre 1 y 500.
 */

public class Ejercicio1 {

	public static void main(String[] args) throws FileNotFoundException {
		try {
			BufferedWriter bw = new BufferedWriter(new FileWriter("primos.dat"));
			for (int i = 2; i < 500; i++) {
				if(esPrimo(i)) {
					bw.write(i+"\n");
				}
			}
			bw.close();
			System.out.println("Se ha creado correctamente.");
		}
		catch(IOException e){
			System.err.println("Fallo");
		}
		
		
	
}

	static boolean esPrimo(int num) {
		for (int i = 2; i < num; i++) {
			if ((num % i) == 0) {
				return false;
			}
		}
		return true;
	}
}

File: Python/ejercicio1Fich/Ejercicio1.py
'''
Escribe un programa que guarde en un fichero con nombre primos.dat los
números primos que hay entre 1 y 500.
'''
# Abre archivo en modo escritura
archivo = open('primos.dat','w')

def esPrimo(num):
    for i in range(2, num):
        if (num%i) == 0:
            return False
    return True



for i in range(2,500):
    if(esPrimo(i)==True):
        archivo.write(str(i)+"\n")
archivo.close()